include/config/config_game.h

```diff
@@ -1,4 +1,19 @@
 #ifndef CONFIG_GAME_H
 #define CONFIG_GAME_H
 
+// Configure text speed (1 is vanilla)
+#define TEXT_SPEED 2
+
+// Rupee counter symbol changes color with wallet upgrades like in MM
+#define WALLET_COLORS
+
+// Wearing bunnyhood has the same functionality as in MM
+#define MM_BUNNYHOOD
+
+// Configure block push speed multiplier
+#define BLOCK_PUSH_SPEED 3
```

Comment:
This should be represented as a float in the define so the user knows that it is one. So, 3.0f, though I would prefer we actually put 1.0f here instead so that the vanilla behavior is the default as the user can choose how fast they want it to be themselves.